frontend-project/src/pages/tags/list/locales/en-US.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+  'tags-list.page-header-content': 'Tags list',
```

Comment:
Tag list (?)
https://english.stackexchange.com/questions/258846/out-of-the-words-users-list-or-users-list-which-is-correct/258851

Reply:
ewentualnie list of users